src/Services/QueuedJobService.php

```diff
@@ -66,6 +66,37 @@ class QueuedJobService
      */
     private static $stall_threshold = 3;
 
+    /**
+     * How early stuck jobs will become eligible for automated retry processing (minutes)
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_sentinel = 1;
+
+    /**
+     * How many jobs are eligible to be automatically retried per one health check
+     * Setting this to 0 disables the job retry feature
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_limit = 0;
```

Comment:
```suggestion
     */
    private static int $job_retry_sentinel = 1;

    /**
     * How many jobs are eligible to be automatically retried per one health check
     * Setting this to 0 disables the job retry feature
     */
    private static int $job_retry_limit = 0;
```
`@config` does nothing without `@internal` and implies there's something weird about these - but since all private static without `@internal` is `@config` automatically we don't need nor want this here.

`@var int` is already part of the explicit type check so this PHPDoc type hint is redundant and at worst could get out of sync if we change the type later.